napalm/base/models.py

```diff
@@ -400,31 +400,6 @@
 
 ConfigDict = TypedDict("ConfigDict", {"running": str, "startup": str, "candidate": str})
 
-TracerouteResultDictEntry = TypedDict(
-    "TracerouteResultDictEntry", {"probes": Dict[int, TracerouteDict]}
-)
-
-OpticsStateDict = TypedDict(
-    "OpticsStateDict", {"instant": float, "avg": float, "min": float, "max": float}
-)
-
-OpticsStatePerChannelDict = TypedDict(
-    "OpticsStatePerChannelDict",
-    {
-        "input_power": OpticsStateDict,
-        "output_power": OpticsStateDict,
-        "laser_bias_current": OpticsStateDict,
-    },
-)
-
-OpticsPerChannelDict = TypedDict(
-    "OpticsPerChannelDict", {"index": int, "state": OpticsStatePerChannelDict}
-)
-
-OpticsPhysicalChannelsDict = TypedDict(
-    "OpticsPhysicalChannelsDict", {"channels": OpticsPerChannelDict}
-)
-
```

Comment:
@itdependsnetworks Why were these ones removed?

Reply:
I introduced these as duplicates within #1476 - when Ken enabled type hinting for that file Mypy caught it and it had to be removed.